### ISSUE TITLE
%bank% placeholder sometimes not parsing correctly

### ISSUE THREAD

**Issue by Maroon28** [silverwolfg11/Pl3xMap-Towny#10]
## Describe the Bug
The bank placeholder doesn't appear to work if the town's name is too long, it seems? It throws this error quite often:
`[02:17:56 ERROR]: [Pl3xMap-Towny] Error applying the replacement %bank% for information on town **The_Floridian_Communist_Ideology!**`

## Expected Behavior

The plugin should parse placeholders without throwing any errors

### Plugin Version

1.0.0-ALPHA-3

### Towny Version
0.97.1.9
### Pl3xMap Version
1.0.0-BETA-154
### Server Software Version
Purpur version git-Purpur-1350 (MC: 1.17.1)

## Additional Comments
This bug only appears to happen with towns with overly long names, never seen it occur for regular towns.

**Comment by silverwolfg11** [comment]
Hi, thanks for reporting this.
Along with the error message, there should be a stack trace (kind of a long message that spits out where the problem occurs in the code) which includes a `Caused by:` line(s). If you could post that stack trace, I would really appreciate it.

**Comment by Maroon28** [comment]
> Hi, thanks for reporting this.
> Along with the error message, there should be a stack trace (kind of a long message that spits out where the problem occurs in the code) which includes a `Caused by:` line(s). If you could post that stack trace, I would really appreciate it.

That was actually the only thing getting spammed, not much else. I would've posted the stacktrace otherwise. Anyways, this issue seems to come from towny itself? Basically what happened was there was a town that _somehow_ bypassed towny's character limit, and the town itself was completely buggered. Ended up renaming the town to something shorter (Communist_Florida), and that fixed all the issues, which is why I closed this issue in the first place. Should've left a comment explaining that though, my bad 😛
